client/state/selectors/get-site-editor-url.js

```diff
@@ -10,14 +10,12 @@ import { getSiteAdminUrl, getSiteSlug } from 'calypso/state/sites/selectors';
  */
 export const getSiteEditorUrl = ( state, siteId ) => {
 	const siteAdminUrl = getSiteAdminUrl( state, siteId );
-
-	if ( ! shouldLoadGutenframe( state, siteId ) ) {
-		return `${ siteAdminUrl }site-editor.php`;
+	const queryArgs = {};
```

Comment:
This made the [fix](https://github.com/Automattic/wp-calypso/pull/76072) gone since the existing query string won't keep after the redirection 🥲